src/components/Dashboard/DashboardChart.vue

```diff
@@ -3,22 +3,22 @@
     <div class="card-content_header">
       <el-row class="card-content_header-row">
         <el-col :span="14">
-          <p>
+          <p class="card-content_header-title">
             {{ portfolioChart.crypto }}
           </p>
         </el-col>
         <el-col :span="10" class="card-content_header-filters">
-          <p
+          <div
             v-for="(value, index) in daysLabels"
             :key="index"
             :class="[portfolioChart.filter !== value ? 'chart_time-filter' : 'chart_time-filter selected']"
             @click="selectLabel(value)">
-            {{ value }}
-          </p>
+            <p class="chart_time-filter_value">{{ value }}</p>
+          </div>
         </el-col>
       </el-row>
     </div>
-    <div class="card-content_body">
+    <div class="card-content_body" :style="{ height: `${windowHeight + 30}px` }">
```

Comment:
I feel that `windowHeight` is a bit confusing because it really is not the window's height.